application/akka/java/AkkaQuickstart.java

```diff
@@ -0,0 +1,22 @@
+package example;
+
+import akka.actor.typed.ActorSystem;
+
+import java.io.IOException;
+public class AkkaQuickstart {
+  public static void main(String[] args) {
+    //#actor-system
```

Comment:
Can these go?

Reply:
They were from template. Now the new code still has them, but they just show what each block of code represents.
If you think it is still redundant, I will remove it.